src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -66,6 +67,23 @@
  * practices for defensive use of serial filters.
  * </strong></p>
  *
+ * <p>The key to disabling deserialization attacks is to prevent instances of
+ * arbitrary classes from being deserialized, thereby preventing the direct or
+ * indirect execution of their methods.
+ * {@link ObjectInputFilter} describes how to use filters and
+ * {@link ObjectInputFilter.Config} describes how to configure the filter and filter factory.
+ * Each stream has an optional deserialization filter
+ * to check the classes and resource limits during deserialization.
+ * The JVM-wide filter factory ensures that a filter can be set on every {@link ObjectInputStream}
+ * and every object read from the stream can be checked.
+ * The {@linkplain #ObjectInputStream() ObjectInputStream constructors} invoke the filter factory
+ * to select the initial filter and it is updated by {@link #setObjectInputFilter}.
+ * <p>
```

Comment:
Strange statement: "The OIS constructors invoke the filter factory to select the initial filter and it is updated by ..."
What about: "The OIS constructors invoke the filter factory to select the initial filter which may be updated / replaced by setObjectInputFilter"